scripts/init-env.sh

```diff
@@ -29,4 +29,9 @@ oc create namespace ${TNF_PARTNER_NAMESPACE} 2>/dev/null
 
 #Partner repo
 export TNF_PARTNER_REPO="${TNF_PARTNER_REPO:-quay.io/testnetworkfunction}"
-export TNF_DEPLOYMENT_TIMEOUT="${TNF_DEPLOYMENT_TIMEOUT:-120s}"
\ No newline at end of file
+export TNF_DEPLOYMENT_TIMEOUT="${TNF_DEPLOYMENT_TIMEOUT:-120s}"
+
+# Detect minikube cluster
+if [[ -z `oc version | grep Server` ]]; then
+    export TNF_MINIKUBE_CLUSTER=1
```

Comment:
Can we give it a name that better reflects what it is? e.g. TNF_OCP_CLUSTER or TNF_NON_OCP_CLUSTER. Also, maybe we should replace the MINIKUBE_ONLY var with this so we don't need to manually set it?

Reply:
Closing PR. Apparently, David's PR https://github.com/test-network-function/cnf-certification-test-partner/pull/51 already defines the IS_MINIKUBE env var for the same purpose.